reader.lua

```diff
@@ -13,6 +13,14 @@ io.stdout:write([[
  [*] Current time: ]], os.date("%x-%X"), "\n")
 io.stdout:flush()
 
+-- Set up ffi search paths
+require("setupkopaths")
+
+-- Apply startup user patches and execute startup user scripts
+local userpatch = require("userpatch")
+userpatch.applyPatches(userpatch.early_once)
```

Comment:
as coded now this will crash the program on devices that return `userpatch` early, without the defined functions (like isAndroid + flavor fdroid)

Reply:
See above: We return `userpatch` with the no-op `applyPatches` and the `userpatch.erlry_once` in any case. So I don't see the crash.